### DISCUSSION TITLE
Whether any support be provided from NetBox during/after implementation in our system.

### DISCUSSION THREAD

**Discussion by athira009** [netbox-community/netbox#14183] [Q&A]
1. Does NetBox provide support during installation in kubernetes cluster?
2. Does NetBox provide maintenance support after installing it in our kubernetes cluster?

**Comment by beevek** [comment]
Hi @athira009 - we (NetBox Labs) offer commercial support only with respect to NetBox Cloud (https://netboxlabs.com/netbox-cloud/).  We are a product company, not a services company, so we have decided not to offer services / support for on-prem installations of NetBox, which enables us to focus instead on building the best product in the market for use cases and customers that can leverage cloud.  That said - we also have a large ecosystem of expert partners who can help with on-prem installations if that is a requirement, and we're happy to connect you with the best partner for your use case.  Please feel free to email me at kbeevers@netboxlabs.com and I'm happy to help.

Best,
-Kris
